public/assets/js/phpci.js

```diff
@@ -14,6 +20,9 @@ var PHPCI = {
             }
         });
 
+        // Setup the date locale
+        moment.locale(PHPCI_LANGUAGE);
```

Comment:
Move this init before document ready (don't change the behavior, but make the code more understandable).